examples/dt/OSASINFRA/DFG-osasinfra-shiftstack_ci-multijob-ocp4.x.md

```diff
@@ -0,0 +1,73 @@
+# DFG-osasinfra-shiftstack_ci-multijob-ocp4.x
```

Comment:
Re-scope this DT for integration pipeline:
- external ceph
- one single compute
- time restriction: 4 hours.

Create another one for the architecture pipeline:
- HCI
- 3 computes
- full testing, time restriction: 5 days.